docs/atari/space_invaders.md

```diff
@@ -56,6 +56,18 @@ zigzaging_bombs=False, fast_bomb=False, invisible_invaders=False)
 
 `invisible_invaders`:  The invaders are invisible, making them more difficult to hit.
 
+### Action Space (Minimal)
+
+In any given turn, an agent can choose from one of 6 actions.
+
+| Action    | Behavior  |
+|:---------:|-----------|
```

Comment:
Looks like this should be similar to pong https://github.com/Farama-Foundation/Multi-Agent-ALE/blob/master/src/games/supported/SpaceInvaders.cpp#L91